src/pages/MusicPage.tsx

```
@@ -24,72 +25,81 @@ const MusicPage: React.FC = () => {
 
-              <div className="mt-16 border-t border-white/5 pt-10">
-                <h2 className="text-xl font-bold mb-6 flex items-center gap-2"><Filter size={18} className="text-primary" /> SOBRE A FAIXA</h2>
-                <div 
-                  className="prose prose-invert max-w-none text-white/60"
-                  dangerouslySetInnerHTML={{ __html: singleTrack.content?.rendered || "" }}
-                />
+                <div className="mt-16 border-t border-white/5 pt-10">
+                  <h2 className="text-xl font-bold mb-6 flex items-center gap-2"><Filter size={18} className="text-primary" /> SOBRE A FAIXA</h2>
+                  <div
+                    className="prose prose-invert max-w-none text-white/60"
+                    dangerouslySetInnerHTML={{ __html: singleTrack.content?.rendered || "" }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)